### ISSUE TITLE
Create Title Screen

### ISSUE THREAD

**Issue by halemaster** [heygamerspigsquadgroup/Cult-Clash#31]
Depends on #6 
- Could just be done as #6 
- Should be able to type in nickname
- When they click 'join game' they should be put into a room
- Possible stretch goal to be able to join friends by having a 'hunt' text box for another nickname
- Options is a stretch goal I think, but we can do it if we want. Not sure what would go in there
